README.org

```diff
@@ -15,22 +15,32 @@ Rate Limit and Throttle WAI Requests
 **** Default (IP address throttler)
      #+BEGIN_SRC haskell
      main = do
-       st <- initThrottler
+       let expirationInterval = TimeSpec 3600 0 -- expire after an hour
+           settings = defaultThrottleSettings expirationInterval
+       th <- initThrottler settings
        let payload  = "{ \"api\", \"return data\" }"
```

Comment:
lol, i guess i didn't realize this  readme was still org-based. I'll convert it to markdown in a follow-up.

Reply:
Yeah, I had to google some stuff but all worked out 🤷‍♂️